camera.py

```diff
@@ -58,7 +58,9 @@ def pan(self, delta: Vector2) -> None:
         self.settings.offset += delta
 
     def rotate(self, delta: Vector2) -> None:
-        self.settings.normalVector += Vector3(delta.x/ 200, delta.y/ 200, 0)
+        speed_factor = 1/50
+        self.settings.normalVector += Vector3(delta.x, delta.y, 0) * speed_factor
```

Comment:
The vector is called `normalVector`, but it's not actually normalized here. Maybe call `self.settings.normalVector.normalize_ip` after adding the delta to normalize the vector in place?

Reply:
A normal vector doesn't necessarily have to be normalized. It is called a normal vector because it's normal to the plane. We do have to normalize though, because we don't want the plane to be stretched out after one rotation.